lib/embed/media_tag.rb

```diff
@@ -84,9 +85,21 @@ def enabled_streaming_sources(file)
       end.join
     end
 
-    def media_element_height
-      return "#{viewer.body_height}px" unless many_primary_files?
-      "#{viewer.body_height.to_i - MEDIA_INDEX_CONTROL_HEIGHT}px"
+    def media_element_height(file)
+      elt_base_height = (file.video_height ? file.video_height : viewer.body_height).to_i
```

Comment:
Could this be more sussinctly written as `elt_base_height = (file.video_height || viewer.body_height).to_i`?